modules/dag-l1/src/test/scala/org/tessellation/dag/l1/infrastructure/trust/storage/TrustStorageSuite.scala

```diff
@@ -0,0 +1,52 @@
+package org.tessellation.dag.l1.infrastructure.trust.storage
+
+import cats.effect.IO
+
+import org.tessellation.dag.l1.config.types.DBConfig
+import org.tessellation.dag.l1.infrastructure.db.Database
+import org.tessellation.scalacheck.generators.trust.genTrustDbValues
+
+import ciris.Secret
+import eu.timepit.refined.auto._
+import weaver._
+
+object TrustStorageSuite extends SimpleIOSuite {
+
+  val testTrustDbValues = genTrustDbValues.sample.get
+
+  test("Test writing trust values to the database - score.") {
+    val dbConfig = DBConfig("org.sqlite.JDBC", "jdbc:sqlite::memory:", "sa", Secret(""))
+    Database.forAsync[IO](dbConfig).use { implicit db =>
+      val trustStorage = TrustDbStorage.make[IO]
+
+      for {
+        _ <- trustStorage.updateTrustValues(Map(testTrustDbValues.peerId -> testTrustDbValues))
+        score <- trustStorage.getTrustScore(testTrustDbValues.peerId)
+      } yield expect.same(score, testTrustDbValues.score)
```

Comment:
```
val peer = testTrustDbValues.peerId
```
can be also moved above.
then, this can be simplified to something like that:
```scala
test("score should be written to the database") { implicit db =>
  trustStorage.updateTrustValues(Map(peer -> testTrustDbValues)) >>
    trustStorage.getTrustScore(peer).flatMap {
      expect.same(_, testTrustDbValues.score)
    }
}
```

Reply:
The tests were simplified as part of the effort to use a shared `Resource`.